src/wallet/bdb.cpp

```diff
@@ -335,7 +335,7 @@ void BerkeleyEnvironment::CheckpointLSN(const std::string& strFile)
 }
 
 
-BerkeleyBatch::BerkeleyBatch(BerkeleyDatabase& database, const char* pszMode, bool fFlushOnCloseIn) : pdb(nullptr), activeTxn(nullptr)
+BerkeleyBatch::BerkeleyBatch(BerkeleyDatabase& database, const char* pszMode, bool fFlushOnCloseIn) : pdb(nullptr), activeTxn(nullptr), m_cursor(nullptr)
```

Comment:
nit, move these to header.

Reply:
I'm pretty sure that won't compile.

Reply:
I mean 04998ef3cb866b2371bb047a5091787f61692588.

Reply:
Ah. Maybe if I have to push again.